pkgs/by-name/om/omnissa-horizon-client/package.nix

```diff
@@ -116,6 +115,22 @@ let
           xorg.libXtst
           zlib
 
+          # c.f. https://github.com/NixOS/nixpkgs/pull/418543
```

Comment:
It's a shame we have to maintain this here, but at least there's precedent, and there doesn't seem to be an appetite to do it globally (https://github.com/NixOS/nixpkgs/pull/396195).

Hopefully this won't stick around for too long.

Reply:
Yes I'm not too fond of this either... My hope is that Omnissa will release a new version that is compatible with the latest libxml2 _eventually_ and we can get rid of that ...